dnsapi/dns_linode.sh

```diff
@@ -68,7 +68,8 @@ dns_linode_rm() {
   _parameters="&DomainID=$_domain_id"
 
   if _rest GET "domain.resource.list" "$_parameters" && [ -n "$response" ]; then
-    response="$(echo "$response" | tr -d "\n" | sed 's/{/\n&/g')"
+    response="$(echo "$response" | tr -d "\n" | sed 's/{/\
```

Comment:
Please replace the '{' char to some other unused char, and then do a sed, and replace the '{' char back to Lf at last.
for example:

```sh
echo "$response" | tr -d "\n"  | tr '{' "_" | sed 's/_/&{/g' | tr "_" "\n"
```